src/main/java/org/carlmontrobotics/commands/AutoMATICALLYGetNote.java

```diff
@@ -54,35 +47,62 @@ public void execute() {
     double forwardDistErrMeters = ll.getDistanceToNoteMeters(); 
     double strafeDistErrMeters = forwardDistErrMeters * Math.tan(angleErrRad);
     // dt.drive(0,0,0);
-    dt.drive(Math.max(forwardDistErrMeters * 2, MIN_MOVEMENT_METERSPSEC),
-        Math.max(strafeDistErrMeters * 2, MIN_MOVEMENT_METERSPSEC), Math.max(angleErrRad * 2, MIN_MOVEMENT_RADSPSEC));
-    // 180deg is about 6.2 rad/sec, min is .5rad/sec
 
-    if (LimelightHelpers.getTV(INTAKE_LL_NAME)) {
-      intake.motorSetIntake(0.5);
-    }
-    if(intake.intakeDetectsNote()) {
-      timer.start();
-    } else {
-      timer.stop();
-      timer.reset();
+    double forwardSpeed = 0;
+    double strafeSpeed = 0;
+    double rotationalSpeed = 0;
+
+    if (LimelightHelpers.getTX(INTAKE_LL_NAME) == 1) {
+      if (angleErrRad >= 0) {
+        forwardSpeed = Math.max(
+            forwardDistErrMeters
+                * SmartDashboard.getNumber("forward speed multiplier", 1.5),
+            MIN_MOVEMENT_METERSPSEC);
+        strafeSpeed = Math.max(
+            strafeDistErrMeters
+                * SmartDashboard.getNumber("strafe speed multiplier", 1.5),
+            MIN_MOVEMENT_METERSPSEC);
```

Comment:
Imo just moving forward and turning simultaneously should be enough to guide the robot to the note. If the robot is automatically correcting the way its facing the note, as long as you drive forward it should still reach the note, strafing might actually mess it up. In this case, if the robot is perfectly facing the note, the robot will still strafe and it might end up missing the note.

Reply:
Ignore this comment, I was thinking if the robot was in ROBOT ORIENTED mode, not FIELD ORIENTED mode.